tools/ci_build/github/azure-pipelines/py-package-build-pipeline.yml

```diff
@@ -22,12 +22,12 @@ parameters:
 - name: python_version
   displayName: 'Python version used for build'
   type: object
-  default: ['3.7', '3.8', '3.9']
+  default: ["3.7", "3.8", "3.9", "3.10.2"]
```

Comment:
The last one should be "3.10"?

Reply:
It's parsed as float and became python 3.1. 
Changed into "'3.10'"